usaspending_api/api_docs/api_documentation/advanced_award_search/spending_over_time.md

```diff
@@ -0,0 +1,63 @@
+## [Spending over Time](#spending-over-time)
+**Route:** `/api/v2/search/spending_over_time/`
+
+**Method:** `POST`
+
+This route takes award filters, and returns spending by time.  The amount of time is denoted by the "group" value.
+
+### Request
+group: how the data is broken up.  ex: `"quarter", "fiscal_year", "month"`
```

Comment:
group: the unit of time that awards are aggregated by. You must use one of: `month`, `quarter`, `fiscal_year`.

Reply:
Done